package.json

```diff
@@ -8,7 +8,7 @@
    "scripts": {
      "build": "babel src -d lib"
    }, 
- "devDependencies": { "babel-cli": "^6.0.0"},
+ "devDependencies": { "babel-cli": "^6.0.0" ,"vis":"0.x.x","jquery":"2.1.3"},
```

Comment:
Don't do this manually, you'll go insane! You can tell NPM to automatically put it in your `package.json` dependencies like so:
```
npm install jquery --save
```